bitcoin/sign.go

```diff
@@ -129,6 +129,85 @@ func (w *BitcoinWallet) buildTx(amount int64, addr btc.Address, feeLevel wi.FeeL
 	return authoredTx.Tx, nil
 }
 
+func (w *BitcoinWallet) buildSpendAllTx(addr btc.Address, feeLevel wi.FeeLevel) (*wire.MsgTx, error) {
+	tx := wire.NewMsgTx(1)
+
+	height, _ := w.ws.ChainTip()
+	utxos, err := w.db.Utxos().GetAll()
+	if err != nil {
+		return nil, err
+	}
+	coinMap := util.GatherCoins(height, utxos, w.ScriptToAddress, w.km.GetKeyForScript)
+
+	totalIn := int64(0)
```

Comment:
I'm concerned that @amangale has already completed the precision work in the multiwallet and that this will conflict. Though I haven't seen the branch from him yet. If the branch is ready (or close) we may need to apply that precision work to the `totalIn` handling.